src/rp2_common/pico_multicore/include/pico/multicore.h

```diff
@@ -24,7 +24,7 @@ extern "C" {
  * \include multicore.c
 */
 
-// PICO_CONFIG: PICO_CORE1_STACK_SIZE, Stack size for core 1, min=0x100, max=0x10000, default=PICO_STACK_SIZE/0x800, group=pico_multicore
+// PICO_CONFIG: PICO_CORE1_STACK_SIZE, Stack size for core 1, min=0x100, max=0x10000, default=PICO_STACK_SIZE (0x800), group=pico_multicore
```

Comment:
I believe the `/` here was indicating "or" rather than "divide" :wink: 
i.e. the default value of PICO_CORE1_STACK_SIZE is PICO_STACK_SIZE if PICO_STACK_SIZE is defined, or 0x800 if PICO_STACK_SIZE isn't defined.

Reply:
correct

Reply:
> I believe the `/` here was indicating "or" rather than "divide" 😉

yes it is indicating an "or", but the symbol is more or less confusing

Reply:
is it ok to use () or should I revert?

Reply:
ping @JamesH65 What syntax does your project-generator expect?

Reply:
According to the resulting tsv file from the source parsing, this... https://github.com/raspberrypi/pico-project-generator/blob/master/pico_configs.tsv

But I like the bracket version better, I wonder if we should allow both formats?

Reply:
> I wonder if we should allow both formats?

I just checked, and the `tools/extract_configs.py` script is happy with either version (and we also have some PICO_CONFIGs with default-values like "1 in debug build 0 otherwise", so spaces are obviously no problem).

> But I like the bracket version better

As a best-of-both-worlds, I think `default=PICO_STACK_SIZE (or 0x800)` looks nice? :thinking: